crates/btc-notify/src/lib.rs

```diff
@@ -0,0 +1,15 @@
+//! # btc-notify
+//!
+//! btc-notify is a crate to deliver real-time notifications on the latest transaction and block
```

Comment:
```suggestion
//! # `btc-notify`
//!
//! `btc-notify` is a crate to deliver real-time notifications on the latest transaction and block
```

Maybe also add that it uses exclusively the ZMQ stuff?

Reply:
I will need to add some RPC stuff for backfilling purposes in a future revision so that won't be accurate.